gimbalsend_node:
  ros__parameters:
    lidarscan_topic: "/lidarscan"
    tracker_topic: "/tracker/send"
    armors_topic: "/detector/armors" 
    gimbal_pub_topic: "/gimbalcontrol"
    max_distance: 8.0
    armor_id: [1, 2, 3, 4, 6]
    detection_timeout: 1.5
    scan_transition_delay: 1.5
